src/components/account/IFXAccountDetail.vue

```diff
@@ -0,0 +1,122 @@
+<script>
+import IFXAccountMixin from '@/components/account/IFXAccountMixin'
+import IFXItemDetailMixin from '@/components/item/IFXItemDetailMixin'
+import IFXItemHistoryDisplay from '@/components/item/IFXItemHistoryDisplay'
+
+export default {
+  name: 'AccountDetail',
+  mixins: [IFXItemDetailMixin, IFXAccountMixin],
+  components: {
+    IFXItemHistoryDisplay,
+  },
+  data() {
+    return {
+      showInvalidAuthorizations: false
+    }
+  },
+  computed: {
+    detailTitle() {
+      if (!this.item.accountType) {
+        return 'Expense Code / PO'
+      }
+      return this.item.accountType
+    }
+  },
+  mounted() {
+    console.log(this.item)
+  }
+}
+</script>
+<template>
+  <v-container v-if="!isLoading">
+    <IFXPageHeader>
+      <template #title>{{ detailTitle }} </template>
+      <template #id>"{{ item.name }}"</template>
+      <template #actions>
+        <span v-if="item.active" class="active-account">
+          <v-icon color="success">check</v-icon> Currently Active
+        </span>
+        <span v-else class="inactive-account">
+          <v-icon>close</v-icon>Currently Inactive
+        </span>
+      </template>
+      <template #content>
+        <IFXItemHistoryDisplay :item="item" />
+      </template>
+    </IFXPageHeader>
+    <v-container px-5 py-0>
+      <v-row justify="start" align="center" dense>
+        <v-col sm="2">
+          <h3>Code</h3>
+        </v-col>
+        <v-col>
+          {{ item.code }}
+        </v-col>
+      </v-row>
+      <v-row justify="start" align="center" dense>
+        <v-col sm="2">
+          <h3>Organization</h3>
+        </v-col>
+        <v-col>
+          {{ item.organization }}
+        </v-col>
+      </v-row>
+      <v-row justify="start" align="center" dense>
+        <v-col sm="2">
+          <h3>Valid From</h3>
+        </v-col>
+        <v-col>
+          {{ item.validFrom | columnDate }}
+        </v-col>
+     </v-row>
+      <v-row justify="start" align="center" dense>
+        <v-col sm="2">
+          <h3>Expiration Date</h3>
+        </v-col>
+        <v-col>
+          {{ item.expirationDate | columnDate }}
+        </v-col>
+      </v-row>
+      <v-row justify="start" align="top" dense>
+        <v-col sm="2">
+          <h3>Authorizations</h3>
+          <div class="invalid" style="margin: 0.5em; font-size: smaller;">Indicates currently inactive authorizations</div>
```

Comment:
Can you use the Vuetify Helper classes here instead of specific inline styles?

Reply:
Would that make things better?